lib/hawkular/operations/operations_api.rb

```diff
@@ -62,39 +62,53 @@ def to_msg_hash
     # @example
     #   Hawkular::Operations::OperationsClient.new(credentials: {username: 'jdoe', password: 'password'})
     def initialize(args)
-      args[:use_secure_connection] ||= false
-      ep = args[:entrypoint]
-
-      unless ep.nil?
-        uri = URI.parse ep.to_s
+      args = {
+        :credentials => {},
+        :options => {},
+        :wait_time => 0.5,
+        :use_secure_connection => false,
+        :entrypoint => nil
+      }.merge(args)
```

Comment:
@josejulio oh, ok, for some reason I thought this project was using the same ruleset as ManageIQ.